src/sentry/incidents/endpoints/project_alert_rule_details.py

```diff
@@ -1,10 +1,12 @@
 from __future__ import absolute_import
 
+from rest_framework import status
 from rest_framework.response import Response
 
 from sentry.api.serializers import serialize
 from sentry.api.serializers.models.alert_rule import AlertRuleSerializer
 from sentry.incidents.endpoints.bases import ProjectAlertRuleEndpoint
+from sentry.incidents.endpoints.serializers import AlertRuleSerializer as DrfAlertRuleSerializer
```

Comment:
UpdateAlertRuleSerializer ?

Reply:
Not sure, this serializer doesn't only support updates, even though we're using it for that in this case...

Reply:
🤷‍♀ What you have is fine.